src/Monolog/Formatter/JsonFormatter.php

```diff
@@ -59,6 +61,38 @@ public function isAppendingNewlines(): bool
         return $this->appendNewline;
     }
 
+    /**
+     * The maximum number of normalization levels to go through
+     */
+    public function getMaxNormalizeDepth(): int
+    {
+        return $this->maxNormalizeDepth;
+    }
+
+    /**
+     * @param int $maxNormalizeDepth
+     */
+    public function setMaxNormalizeDepth(int $maxNormalizeDepth)
+    {
+        $this->maxNormalizeDepth = $maxNormalizeDepth;
+    }
+
+    /**
+     * The maximum number of items to normalize per level
+     */
+    public function getMaxNormalizeItemCount(): int
+    {
+        return $this->maxNormalizeItemCount;
+    }
+
+    /**
+     * @param $maxNormalizeItemCount
+     */
+    public function setMaxNormalizeItemCount($maxNormalizeItemCount)
```

Comment:
Why no int type hint here? Also please don't include @param annotations for parameters with type hints, they're redundant.

Reply:
Alright. Since the code isn't too consistent, I didn't know what to go with. I also see that this is 7.1 and not 7.0, so I'll add the void return type as well.